src/main/java/com/iexec/worker/feign/CustomCoreFeignClient.java

```diff
@@ -145,6 +145,7 @@ public TaskNotificationType updateReplicateStatus(String chainTaskId, ReplicateS
                 coreClient.updateReplicateStatus((String) args.get(JWTOKEN), (String) args.get("chainTaskId"),
                         (ReplicateStatusUpdate) args.get("statusUpdate"));
 
+        // As long as the Core doesn't reply, we try to contact it. It may be rebooting.
```

Comment:
Comment to move to while loop I think

Reply:
Good catch! Please see https://github.com/iExecBlockchainComputing/iexec-worker/pull/483/commits/ac0fba34ef042527ac3cf94846929d6b2f5959fd.